### DISCUSSION TITLE
Disable logging of sensitive params

### DISCUSSION THREAD

**Discussion by henryaj** [excid3/noticed#334] [General]
Is there any way to disable logging of sensitive params?

Whenever noticed enqueues a job, I can see all the params of that job are printed to STDOUT. It looks like ActiveJob supports selectively disabling these by [setting `self.log_arguments = false` on the job](https://github.com/rails/rails/pull/37660/files); is there a similar feature here? Thanks!

**Comment by excid3** [comment]
Nothing currently, but PRs are welcome. The trouble is we save the params in the database so I'm not sure it's really solvable unless you encrypt them.

**Comment by henryaj** [comment]
Ah, I meant "logging" specifically in terms of "writing sensitive data to logs" rather than the storing of said data, which I'm fine with.

Great, might look into a PR!

**Comment by excid3** [comment]
We don't do any custom logging. Just hand things off to the database an ActiveJob. So I'd be curious to see which part is writing to the logs then if you've turned them off for ActiveJob. Probably the ActiveRecord logs would be the only other thing?
